lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -2366,6 +2366,28 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                                                         exp);
         return success();
       });
+  patterns.onOp(
+      "Softsign", 22, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        Value input;
+        if (binder.tensorOperand(input) ||
+            binder.tensorResultType(resultType)) {
+          return failure();
+        }
+
+        Value abs_x = rewriter.create<Torch::AtenAbsOp>(binder.getLoc(),
```

Comment:
nit: mlir uses `thisCase` for the local variable naming convention
```suggestion
        Value absX = rewriter.create<Torch::AtenAbsOp>(binder.getLoc(),
```